index.js

```diff
@@ -11,6 +11,11 @@ module.exports = (grunt, options = {}) => {
 
 	let cwd = process.cwd();
 	let config = options.config || pkgUp.sync();
+
+	if (config === null) {
+		console.error('\u001B[41m%s\u001B[0m', 'ABORTED: package.json not found.\nPlease be sure to create a package.json and install all dependecies before run a grunt task.');
```

Comment:
You should use `grunt.fail` to log errors and exit, see: https://gruntjs.com/api/grunt.fail

Reply:
Moreover, by doing so, you don't need the error ansi codes at all.

Reply:
Thank you for the suggestion. I will work on that right away.